readme.md

```diff
@@ -1097,3 +1098,153 @@ skip-csproj-packagereference: true
 ```
 
 </details>
+
+## Mgmt plane configurations
+
+The mgmt .Net SDK requires the resources to be generated hierarchical, therefore mgmt generator makes quite a few modification on the code model inout from the modelerfour. To better recognize the hierarchical structure of resources, the mgmt generator will generate everything from the point of view of request paths, instead of operation groups. Therefore quite a few new configurations are introduced to tweak the behavior how we identify the resource classes as well as the hierarchy.
```

Comment:
What does `inout from the modelerfour` mean?

Reply:
Should be `input` instead of `inout` it is a typo